src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -229,6 +229,40 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
                 }
             },
 
+            outerHTML: {
+                condition: domUtils.isElementNode,
+                get:       el => cleanUpHtml(el.outerHTML, el.parentNode && el.parentNode.tagName),
+
+                set: (el, value) => {
+                    var parentEl = el.parentNode;
+
+                    if (parentEl) {
+                        var parentDocument = domUtils.findDocument(parentEl);
+                        var parentWindow   = parentDocument ? parentDocument.defaultView : null;
+
+                        el.outerHTML = processHtml('' + value, parentEl.tagName);
+
+                        // NOTE: For the iframe with an empty src.
+                        if (parentWindow && parentWindow !== window &&
+                            parentWindow[INTERNAL_PROPS.processDomMethodName])
+                            parentWindow[INTERNAL_PROPS.processDomMethodName](parentEl, parentDocument);
+                        else if (window[INTERNAL_PROPS.processDomMethodName])
+                            window[INTERNAL_PROPS.processDomMethodName](parentEl);
+
+                        // NOTE: This check is required for an element in an unavailable window.
+                        // NOTE: Use timeout, so that changes take effect.
+                        if (window.self && domUtils.isBodyElement(el))
```

Comment:
You can't use `el` after `el.outerHTML` was executed (in Chrome, at least), because:
```js
var body = document.body;

body.outerHTML;
//<body>BODY1</body>

body.outerHTML = '<body>BODY2</body>'
//"<body>BODY2</body>"

body.outerHTML
//"<body>BODY1</body>"

body.parentNode
//null

document.body
//<body>BODY2</body>
```
At least for `<body>`, rewiting `el.outerHTML` leads to behavior like `el.parentNode.replaceChild(newElFromOuterHTML, el)`, and `el` points to original element which HTML is not changed and that is removed from DOM.